src/test/java/org/apache/commons/io/file/PathUtilsContentEqualsTest.java

```diff
@@ -44,7 +44,7 @@ private String getName() {
 
     @Test
     public void testDirectoryAndFileContentEquals() throws Exception {
-        // Non-existent files
+        // Nonexistent files
```

Comment:
Let's not change this please, this is not a spelling error, both spellings are acceptable, one is clearer due to the hyphen.

Reply:
Sure, but do note that you're already using it in [‎src/main/java/org/apache/commons/io/FileUtils.java](https://github.com/apache/commons-io/blob/06fde31494c279ad940149e1a3d4944040c73c0d/src/main/java/org/apache/commons/io/FileUtils.java#L1091) per https://github.com/search?q=repo%3Aapache%2Fcommons-io%20nonexistent&type=code

Reply:
I've dropped those changes, but I've thrown in some more `existant` -> `existent` changes.

Reply:
Thank you. I'd rather normalize that word on the hyphenated variant.

Reply:
Sorry, how would you do that? consider https://github.com/apache/commons-io/compare/318645fd97d6c31cfbce6f5a7fa83d5350e87708..810cf30344db9d3e5fffc8ea074d50fae136e558

https://github.com/apache/commons-io/blob/810cf30344db9d3e5fffc8ea074d50fae136e558/src/test/java/org/apache/commons/io/file/PathUtilsTest.java#L102

Reply:
Err... this is a PR, you edit the files?

Reply:
Then I think this PR has what you want.

Reply:
It is not. Local vars should be "nonexistentFoo" -> "nonExistentFoo".

Reply:
ok, I think this addresses your request